imports/plugins/core/checkout/client/components/addEmail.js

```diff
@@ -0,0 +1,92 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { Meteor } from "meteor/meteor";
+import { Orders } from "/lib/collections";
+import { Reaction, i18next } from "/client/api";
+import { Components } from "@reactioncommerce/reaction-components";
+
```

Comment:
We should be adding docs to all new components: https://github.com/reactioncommerce/reaction-jsdoc#documenting-react-components